src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, FieldArray } from 'redux-form';
+import moment from 'moment';
+
+import { Collapsible } from '@edx/paragon';
+import RenderInputTextField from '../RenderInputTextField';
+import RenderSelectField from '../RenderSelectField';
+import TranscriptLanguage from './TranscriptLanguage';
+
+const formatCourseRunTitle = (courseRun) => {
+  if (courseRun) {
+    const startDateString = moment(courseRun.start).format('MMM Do YYYY');
+    const pacingTypeString = courseRun.pacing_type.split('_').map(pacingType =>
```

Comment:
This line is a chore to parse. But it's fine™️

Reply:
Will add a comment, but it seemed better than doing a switch(pacing_type): case 'self_paced':